src/base/bittorrent/peerinfo.cpp

```diff
@@ -246,8 +246,8 @@ QString PeerInfo::connectionType() const
 
 void PeerInfo::calcRelevance(const TorrentHandle *torrent)
 {
-    const QBitArray &allPieces = torrent->pieces();
```

Comment:
Why did you change all the similar code?

Reply:
I don't know what you mean?

Reply:
This one and the following similar lines are correct. Why did you change them? I don't see any description of these changes neither in commit message nor in PR comment.

Reply:
`torrent->pieces();` returns `QBitArray`, personally I will prefer using `const QBitArray allPieces` to store it instead of `const QBitArray &allPieces`.

Reply:
>  I don't see any description of these changes neither in commit message nor in PR comment.

The description is in the 1st commit message.  However, @Chocobo1 mentions RVO, and I'm puzzled what is meant by that. Obviously, if the code was working with a reference, there is no reference to a temporary object returned. Thus how does the RVO relate here?

Reply:
>Thus how does the RVO relate here?

In the new code the RVO may kick in... how should I reword the commit msg? Or maybe just remove the last sentence?

Reply:
>The description is in the 1st commit message. 

Oops... I just didn't notice it. Two commits have the same messages (if they were more meaningful, they are unlikely to be the same).

>However, @Chocobo1 mentions RVO, and I'm puzzled what is meant by that. 

I suppose @Chocobo1 mean RVO (or NRVO) can be used by the compiler when the result will be assigned to variable instead of reference.

Reply:
I still don't get it. If the old code is plainly wrong (a reference to a temporary), why not write about that directly?

Reply:
The previous comments is ambiguous. I wanted to point out that there absolutely no difference between the old one and the new one. Hence why change that?

Reply:
>there absolutely no difference between the old one and the new one. Hence why change that?

Maybe to make all the code using the same style?

Reply:
>how should I reword the commit msg? Or maybe just remove the last sentence?

I've updated the commit messages.

Reply:
Could you, please, correct it one more time to replace "binding reference" with "binding constant reference", please? Because the former one reads as if a mistake was fixed.

Reply:
done.

Reply:
thanks!